adlfs/lib.py

```diff
@@ -245,16 +259,25 @@ def call(self, op, path='', **kwargs):
         func = getattr(requests, method)
         url = self.url + path
         try:
+            self.head['x-ms-client-request-id'] = str(uuid.uuid4())
```

Comment:
I have no issue switching to uuid1. By default, I tend to use uuid4 since it's anonymous and doesn't rely on the hostname and current time.